tests/integration/conftest.py

```diff
@@ -6,13 +6,22 @@
 import socket
 import tempfile
 import time
+from functools import partialmethod
 from pathlib import Path
 
 import docker
+import fabric
 import pytest
 from docker.models.containers import Container
 
 
+@pytest.fixture(autouse=True)
+def mock_fabric_run(monkeypatch):
+    monkeypatch.setattr(
+        fabric.Connection, "run", partialmethod(fabric.Connection.run, in_stream=False)
+    )
```

Comment:
Okay, this seems to remedy the issue around the `jfr project check` test failing when pytest is capturing output.